service/topology_state_machine.hh

```diff
@@ -120,6 +120,7 @@ struct topology {
         left_token_ring,
         rollback_to_normal,
         truncate_table,
+        lock, // Blocks the state machine until externally changed.
```

Comment:
If it's intended only for testing, let's document it as such

Reply:
Ok